src/modules/universal-discovery/components/create/create.choose.content.type.component.js

```diff
@@ -7,19 +7,50 @@ export default class ChooseContentTypeComponent extends Component {
     constructor(props) {
         super(props);
 
```

Comment:
The file name should reflect the class name.

Reply:
Changing the class name or file name is kind of a BC break now. We want to release it in 2.1.x. I'm not sure if we should do this.